R/dynamic.R

```diff
@@ -279,37 +287,48 @@ obsolete_out = function(input, output, pattern = '[.]Rmd$', outext = '.md') {
   if (any(idx)) cbind(src[idx], out[idx])
 }
 
-build_jekyll = function(input, output, ...) {
+build_jekyll = function(input, output, in_session = FALSE, verbose = TRUE, ...) {
+  # extract arguments in `...` and coerce to string
+  params = as.list(match.call())[-1]
+  idx = !names(params) %in% names(formals(build_jekyll))
+  dots = params[idx]
+  dotz = paste(names(dots), as.character(dots), sep = "=", collapse = ", ")
```

Comment:
I suppose this bit is unnecessary, but if this is ever explicitly exposed to users, it could be useful to pass arguments to `knitr::knit()`